opentelemetry-api/src/opentelemetry/attributes/__init__.py

```diff
@@ -129,10 +148,10 @@ def __init__(
                 )
         self.maxlen = maxlen
         self.dropped = 0
+        self.max_value_len = max_value_len
```

Comment:
Might need some validation for: `Empty value is treated as infinity. Non-integer and negative values are invalid.`

Reply:
I know this is a small ask but I ended up fixing too many unrelated issues in this PR. Created an issue for this and will take care of it in a separate one tomorrow. https://github.com/open-telemetry/opentelemetry-python/issues/2052